OakCityV2/DetailViewController.h

```diff
@@ -14,9 +14,8 @@
 @property (weak, nonatomic) NSString *company;
 @property (weak, nonatomic) NSString *desc;
 @property (weak, nonatomic) NSString *url;
```

Comment:
Instead of passing these fields individually, you should pass your Job object instead.  The point of the Job object is to act as a container for all these data fields so that you can pass them around as one object.

The DetailViewController always needs to hold on to the job object, so it should be a strong reference.